src/main/resources/edu/cornell/kfs/paymentworks/businessobject/datadictionary/PaymentWorksVendor.xml

```diff
@@ -194,13 +196,45 @@
 		</property>
 	</bean>
 
+	<bean id="PaymentWorksVendor-requestingCompanyLegalFirstName" parent="PaymentWorksVendor-requestingCompanyLegalFirstName-parentBean" />
+    <bean id="PaymentWorksVendor-requestingCompanyLegalFirstName-parentBean" abstract="true" parent="AttributeDefinition">
+        <property name="name" value="requestingCompanyLegalFirstName" />
+        <property name="label" value="Requesting Company Legal First Name" />
+        <property name="forceUppercase" value="false" />
+        <property name="shortLabel" value="Vendor First Name" />
+        <property name="maxLength" value="50" />
+        <property name="validationPattern">
+            <bean parent="UTF8AnyCharacterValidationPattern"
+                p:allowWhitespace="true" />
+        </property>
+        <property name="control">
+            <bean parent="TextControlDefinition" p:size="45" />
```

Comment:
I spoke with Nancy about this.  The text control field size, won't prevent the max length from being achieved.  It may affect how it is displayed on screen.  However, since we don't have "good data" yet, we can't see how the display of a large of data will look.  These text controls can be adjusted later as needed.